src/flamegpu/simulation/CUDAEnsemble.cu

```diff
@@ -417,6 +764,13 @@ int CUDAEnsemble::checkArgs(int argc, const char** argv) {
         if (arg.compare("--standby") == 0) {
 #ifdef _MSC_VER
             config.block_standby = false;
+#endif
+            continue;
+        }
+        // --no-mpi Disable MPI support
+        if (arg.compare("--no-mpi") == 0) {
```

Comment:
I don't think `--no-mpi` is neccesary Just don't run with `mpirun -n <N>` with any value other than 1 in this case?